File: 19JSP04/src/lap1/day3/Employee.java
package lap1.day3;

public class Employee {
	int id;
	String name;
	double salary;
	int age;
	boolean married;
	Job job;
	public Employee() {
		
	}
	public Employee( int id, String name, double salary, Job jobName)
	{
		this.id = id;
		this.name = name;
		this.salary = salary;
		job = jobName;
	}
}

File: 19JSP04/bin/c#/New folder (3)/employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmployeeManagement
{
    abstract class employee
    {
        public String name { get; set; }
        public int age { get; set; }
        public employee() { }
        public employee(String name, int age)
        {
            this.name = name;
            this.age = age;
        }
        public abstract double salary();
        public virtual double thuong()
        {
            return 1000;
        }
        public virtual double heSoLuong()
        {
            return 3000;
        }
    }
}
